elastic/kibana
# [Discover] Align time filters creation in `Test query` of alert rule with Discover main

## Summary

Fixes #145821

This PR adjusts units for time filter creation of `Test query` alert function to be the same with Discover main `fetch_all()`.

### Test notes
- Open Discover and apply some filter
- Select some specific time period
- Open Alerts flyout and select the same time window
- Click Test query
- Number of matched documents should be the same with Discover total hits number.


### Checklist
